ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTlsMisconfigurationTest.java

```diff
@@ -76,10 +76,9 @@
   private static final String CLIENT_VERSION = "TestClientVersion/0.1.0";
   private static final BigInteger CHAIN_ID = BigInteger.valueOf(123);
   private static final Collection<RpcApi> JSON_RPC_APIS = List.of(ETH, NET, WEB3);
-  private static final String KEYSTORE_RESOURCE = "JsonRpcHttpService/rpc_keystore.pfx";
-  private static final String KNOWN_CLIENTS_RESOURCE = "JsonRpcHttpService/rpc_known_clients.txt";
   private static final NatService natService = new NatService(Optional.empty());
-
+  private final SelfSignedP12Certificate besuCertificate = SelfSignedP12Certificate.create();
```

Comment:
nit: is besuCertificate correct here, or is this the certificate presented by the client?